src/Serializer/BaseNormalizer.php

```diff
@@ -11,6 +11,13 @@ abstract class BaseNormalizer extends AbstractNormalizer {
 
   use ClassNameSupportNormalizerTrait;
 
+  /**
+   * {@inheritdoc}
+   */
+  public function denormalize($data, $class, $format = NULL, array $context = []) {
+    throw new \RuntimeException("Method not implemented.");
```

Comment:
Do we need this? DenormalizerInterface contains the method... if we don't implement, won't it bork anyway?

Reply:
Just means you can have a normalizer that doesn't support denormalization.